cli/src/ui/service_methods.rs

```diff
@@ -11,99 +11,77 @@
 use super::console::{Icon, StyledTable};
 use crate::app::UiConfig;
 
-use restate_meta_rest_model::services::{InstanceType, MethodMetadata};
-
 use comfy_table::{Cell, Color, Table};
+use restate_meta_rest_model::components::ComponentType;
+use restate_meta_rest_model::handlers::HandlerMetadata;
 
-pub fn create_service_methods_table(ui_config: &UiConfig, methods: &[MethodMetadata]) -> Table {
+pub fn create_component_handlers_table(
+    ui_config: &UiConfig,
+    handlers: &[HandlerMetadata],
+) -> Table {
     let mut table = Table::new_styled(ui_config);
     table.set_styled_header(vec!["METHOD", "INPUT TYPE", "OUTPUT TYPE"]);
 
-    for method in methods {
+    for handler in handlers {
         table.add_row(vec![
-            Cell::new(&method.name),
-            Cell::new(&method.input_type),
-            Cell::new(&method.output_type),
+            Cell::new(&handler.name),
+            Cell::new(handler.input_description.as_deref().unwrap_or("any")),
+            Cell::new(handler.output_description.as_deref().unwrap_or("any")),
         ]);
     }
     table
 }
 
 pub fn create_service_methods_table_diff(
```

Comment:
Should that be called `create_component_handlers_table_diff` to be consistent in our terminology?